README.md

```diff
@@ -287,6 +287,10 @@ If set to false, doesn't allocate a TTY. This is useful in some situations where
 
 Default: `true` for Linux and macOS, and `false` for Windows.
 
+### `interactive` (optional, boolean)
+
+If set to false, doesn't connect `stdin` to the process. Some scripts fall back to asking for user input in case of errors, if the process has `stdin` connected and this results in the process waiting for input indefinitely.
+
 ### `user` (optional, string)
```

Comment:
Not clear which one is the default. Looking at the code it seems like it's false.

In that case, a clearer wording would be "If set to true, connect to `stdin` of the process" or so.

Reply:
The default is true, so the wording is probably correct. The wording is based on a sibling parameter `tty`